src/component/handlers/drag/DraftEditorDragHandler.js

```diff
@@ -23,6 +23,7 @@ const getUpdatedSelectionState = require('getUpdatedSelectionState');
 const nullthrows = require('nullthrows');
 
 import type SelectionState from 'SelectionState';
+const isEventHandled = require('isEventHandled');
```

Comment:
why not `import {isEventHandled} from 'isEventHandled';`?

I assumed that import was now preferred over require and using that now as we go.